guides/Overview.md

```diff
@@ -1,9 +1,19 @@
 # Introduction
 
-Velo's APIs (previously Corvid) empower you to take full control of your site’s functionality. Use the APIs to interact with site elements, your site’s database content, Wix Apps, and external services. The APIs also give you access to information about your site, its users, and more.
+Welcome to Wix's Velo API Reference. This reference is your comprehensive guide to using Velo, Wix's powerful JavaScript-based development platform, to create dynamic and interactive web experiences. Velo empowers you to take full control of your site’s functionality. Use Velo APIs to fully customize your site, from frontend design to backend functionality.
```

Comment:
```suggestion
Welcome to Wix's Velo API Reference. This reference is your comprehensive guide to using Velo, Wix's powerful JavaScript-based development platform, to create dynamic and interactive web experiences. Velo empowers you to take full control of your site, from frontend design to backend functionality.
```
This is a strong intro but it felt to me like the last two sentences partially repeat the same point. This is one way to combine them but you could also choose another, like "Velo empowers you to fully customize your site, from frontend design to backend functionality."